tests/ui/specialization/ctfe/default-assoc-type.rs

```diff
@@ -0,0 +1,29 @@
+// ICE fixed by #119821.
```

Comment:
same here https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=a01dd24080ed88b3a4249714dcb955c0

Reply:
also still passes after this PR